packages/gatsby/cache-dir/loader.js

```diff
@@ -324,32 +376,34 @@ export class BaseLoader {
   }
 
   loadAppData(retries = 0) {
-    return doFetch(`${__PATH_PREFIX__}/page-data/app-data.json`).then(req => {
-      const { status, responseText } = req
+    return this.memoizedGet(`${__PATH_PREFIX__}/page-data/app-data.json`).then(
```

Comment:
Do we need path-prefix everywhere then? A bit confused how it works.

Reply:
Yup, need this always. It's set to an empty string if there is no prefix.

Reply:
I meant, what about the other cases where we fetch, do we need that theer too?